CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing to OSRD
+
+## Where to start ?
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+If you are brand new to OSRD or open-source development, we recommend chatting with us on [libera chat](https://web.libera.chat/#osrd) or going through the [GitHub issues](https://github.com/DGEXSolutions/osrd/issues) tab to find issues that interest you. There are number of issues listed under `good first issue` where you could start out.
+
+## Contribute code
+
+Commit style is detailed in the [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code/#git-commit-style).
+
+### Pull request / code review
+
+- Every PR, good or bad, is an act of generosity.
+  Opening with a positive comment will help the author feel rewarded, and your subsequent remarks may be heard more clearly.
+  You may feel good also.
```

Comment:
Seems weird to address in the same doc emotions: "generosity" "feel good" and technical, "how to", details . I would delete these 3 lines (especially as the file is already quite long)

Reply:
duplicate comment as https://github.com/DGEXSolutions/osrd/pull/3347#discussion_r1118789794, I will remove as you are a lot to ask for the removal of this.